core/tests/Test.Microsoft.Identity.Core.UIAutomation/UserInformationFieldIds.cs

```diff
@@ -51,6 +51,11 @@ public void DetermineFieldIds(LabUser user)
                         break;
                 }
             }
+            else if (user.UserType == UserType.B2C)
```

Comment:
Seems a bit convoluted. I propose to consolidate with Federated etc., so I would create an Authority enum with { AdfsV3, AdfsV4, B2C_Local etc. Then properties like IsFederated do not even need to exist, or if they do they would jus check if authority is ADFSv3 or ADFSv4.

Reply:
we synced off-line about this. going to leave it for now and talk w/lab about possible changes on their end before we modify things here.

---
In reply to: [232941462](https://github.com/AzureAD/azure-activedirectory-library-for-dotnet/pull/1403#discussion_r232941462) [](ancestors = 232941462)